spec/requests/barcode_gets_spec.rb

```diff
@@ -3,18 +3,16 @@
 
 RSpec.describe 'Barcode Gets', type: :request do
   describe 'GET /barcode/32101070300312' do
-    it 'returns a collection of bibliographic records' do
-      pending 'Replace with Alma'
-      # stub_voyager_barcodes('32101070300312')
+    it 'returns a collection of bibliographic records', pending: 'Replace with Alma' do
```

Comment:
The pending specs in this file need to be reviewed and updated. I will create a ticket